src/crud2 copy.js

```diff
@@ -0,0 +1,110 @@
+import { clearing, clear } from './clear.js';
+
+const taskListDisplay = document.querySelector('#data');
+const inputValue = document.querySelector('.new-task');
+const addButton = document.querySelector('.add-to-list');
+const deleteBtn = document.querySelector('.delete');
```

Comment:
1. On line 6 you could have used deltebutton instead of deletebtn in order to follow JS best practices to declared self explanatory variables.

Reply:
Updated line 6, as requested! 👍